DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -21,9 +22,25 @@ public IActionResult Index()
             return View();
         }
 
-        public IActionResult SignIn()
+        [HttpPost]
+        [Route("/Login")]

```

Comment:
Do we need the route?

Reply:
I thought it did in order to get rid of the error summary anchors (e.g. #Username) from the url when resubmitting after clicking one of those links. After retesting, it looks like it is unnecessary (provided the method is renamed Index so the url is not /Login/Login. So I shall remove the route and change it the name to Index.